src/regtech_user_fi_management/entities/engine/engine.py

```diff
@@ -1,16 +1,13 @@
 from sqlalchemy import create_engine
-from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.orm import sessionmaker
 from regtech_user_fi_management.config import settings
 
 engine = create_engine(str(settings.inst_conn), echo=settings.db_logging).execution_options(
     schema_translate_map={None: settings.inst_db_schema}
 )
-SessionLocal = scoped_session(sessionmaker(engine, expire_on_commit=False))
+SessionLocal = Session = sessionmaker(engine, expire_on_commit=False)
```

Comment:
adding the `Session` name as a more syntactically correct term since it's not thread local anymore; clean up to remove SessionLocal later after downstream repos use the new term